### ISSUE TITLE
Plus sign on codebases section on "Analyze" page redirects to wrong page (first few seconds after page load)

### ISSUE THREAD

**Issue by rhopp** [openshiftio/openshift.io#3489]
First few seconds after loading "Analyze" page, the plus sign on "Codebases" section redirects to `https://openshift.io/<username>/<spaceName>/create/(action:add-codebase)` instead of using new import application wizard.

From this screencast it's apparent. First clicking on the plus sign causes new import application wizard to show (correct behavior). The second click on the plus is immediately after refreshing that page and i redirects user to wrong page:
![peek 2018-05-07 12-36](https://user-images.githubusercontent.com/1215011/39698181-175a7eee-51f4-11e8-8d3b-ab2148b5b234.gif)

**Comment by rhopp** [comment]
@arunkumars08 Basically yes... In the first few seconds after page load, the plus sign redirects to wrong page. After those few seconds, I guess that some script finishes loading and the plus sign behaves as expected (probably onClick method on that element is overriden?)
@aptmac This is happening in both "Production Only features" and "Internal Experimental". If you wan't me to try that with other levels too, let me know, but I guess the result will be still the same.

**Comment by aptmac** [comment]
@rhopp Thanks for the heads up! I was just able to reproduce it.

This issue should be fixed in a commit [0] I made the other day to fix the header highlighting when using the add-codebase action, and I'm pretty sure this code hasn't made it to production yet. In local runs I'm not able to reproduce the bug, but in prod I am. Essentially all instances of the codebases slider has been replaced with the new app launcher, so I'll keep an eye on this issue. 

[0] https://github.com/fabric8-ui/fabric8-ui/commit/04b114d89a48f7810d1fd8ea9c805a7ea8e0205c#diff-e7e906111c620d447f282025221ac240
